Integrations/AzureSecurityCenter/AzureSecurityCenter.py

```diff
@@ -0,0 +1,1253 @@
+    response = initiate_jit(resource_group_name, asc_location, policy_name, vm_id, port, source_address, duration)
+    policy_id = "/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Security/locations/{}/jitNetworkAccessPolicies/{}".format(
+        SUBSCRIPTION_ID, resource_group_name, asc_location, policy_name)
+    virtual_machines = response.get('virtualMachines')
+    if virtual_machines and len(virtual_machines) > 0:
+        machine = virtual_machines[0]
+        port = machine.get('ports')[0]
+
+        outputs = {
+            'VmID': machine.get('id'),
+            'PortNum': port.get('number'),
+            'AllowedSourceAddress': port.get('allowedSourceAddressPrefix'),
+            'EndTimeUtc': port.get('endTimeUtc'),
+            'Status': port.get('status'),
+            'Requestor': response.get('requestor'),
+            'PolicyID': policy_id
+        }
+
+        md = tableToMarkdown(
+            'Azure Security Center - Initiate JIT Access Request',
+            outputs,
+            [
+                'VmID',
+                'PortNum',
+                'AllowedSourceAddress',
+                'EndTimeUtc',
+                'Status',
+                'Requestor'
+            ],
+            removeNull=True
+        )
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'Contents': response,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': md,
+            'EntryContext': {
+                'AzureSecurityCenter.JITPolicy(val.ID && val.ID === obj.{}).Initiate(val.endTimeUtc === obj.EndTimeUtc)'.format(
+                    policy_id): outputs}
+        })
+
+
+def initiate_jit(resource_group_name, asc_location, policy_name, vm_id, port, source_address, duration):
+    cmd_url = '/resourceGroups/{}/providers/Microsoft.Security/locations/{}/jitNetworkAccessPolicies/{}/initiate?api-version={}'.format(
+        resource_group_name, asc_location, policy_name, JIT_API_VERSION)
+    # only supports init access for one vm and one port now
+    data = {
+        "virtualMachines": [{
+            "ID": vm_id,
+            "ports": [{
+                "number": port,
+                "duration": duration,
+                "allowedSourceAddressPrefix": source_address
+            }]
+        }]
+    }
+    response = http_request('POST', cmd_url, body=data)
+    return response
+
+
+def delete_jit_command(args):
+    asc_location = args.get('asc_location')
+    resource_group_name = args.get('resource_group_name')
+    policy_name = args.get('policy_name')
+    delete_jit(asc_location, resource_group_name, policy_name)
+
+    policy_id = "/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Security/locations/{}/jitNetworkAccessPolicies/{}".format(
+        SUBSCRIPTION_ID, resource_group_name, asc_location, policy_name)
+    outputs = {
+        "ID": policy_id,
+        "Action": 'deleted'
```

Comment:
is this correlated with the response of other commands?

Reply:
please explain